developerDocs/getting-started.md

```diff
@@ -0,0 +1,320 @@
+
+```TypeScript
+/**
+   * Attrs used by orderbook to make queries easier
+   * More to come soon!
+   */
+  side: "bid" | "ask", // "bid" for buy orders, "ask" for sell orders
+  protocol?: "seaport"; // Protocol of the order (more options may be added in future)
+  maker?: string, // Address of the order's creator
+  taker?: string, // The null address if anyone is allowed to take the order
+  owner?: string, // Address of owner of the order's item
+  sale_kind?: SaleKind, // 0 for fixed-price, 1 for Dutch auctions
+  assetContractAddress?: string, // Contract address for order's item
+  paymentTokenAddress?: string; // Contract address for order's payment token
+  tokenId?: number | string,
+  tokenIds?: Array<number | string>,
+  listedAfter?: number | string, // This means listing_time > value in seconds
+  listedBefore?: number | string, // This means listing_time <= value in seconds
+  orderBy?: "created_date" | "eth_price", // Field to sort results by
+  orderDirection?: "asc" | "desc", // Sort direction of orderBy sorting of results
+  onlyEnglish?: boolean, // Only return english auction orders
+
+  // For pagination
+  limit?: number,
+  offset?: number,
+```
+
+### Buying Items
+
+To buy an item, you need to **fulfill a sell order**. To do that, it's just one call:
+
+```typescript
+const order = await openseaSDK.api.getOrder({ side: "ask", ... })
+const accountAddress = "0x..." // The buyer's wallet address, also the taker
+const transactionHash = await openseaSDK.fulfillOrder({ order, accountAddress })
+```
+
+Note that the `fulfillOrder` promise resolves when the transaction has been confirmed and mined to the blockchain. To get the transaction hash before this happens, add an event listener (see [Listening to Events](#listening-to-events)) for the `TransactionCreated` event.
+
+If the order is a sell order (`order.side === "ask"`), the taker is the _buyer_ and this will prompt the buyer to pay for the item(s).
+
+### Accepting Offers
+
+Similar to fulfilling sell orders above, you need to fulfill a buy order on an item you own to receive the tokens in the offer.
+
+```typescript
+const order = await openseaSDK.api.getOrder({ side: "bid", ... })
+const accountAddress = "0x..." // The owner's wallet address, also the taker
+await openseaSDK.fulfillOrder({ order, accountAddress })
+```
+
+If the order is a buy order (`order.side === "bid"`), then the taker is the _owner_ and this will prompt the owner to exchange their item(s) for whatever is being offered in return. See [Listening to Events](#listening-to-events) below to respond to the setup transactions that occur the first time a user accepts a bid.
+
+### Transferring Items or Coins (Gifting)
+
+A handy feature in OpenSea.js is the ability to transfer any supported asset (fungible or non-fungible tokens) in one line of JavaScript.
+
+To transfer an ERC-721 asset or an ERC-1155 asset, it's just one call:
+
+```typescript
+const transactionHash = await openseaSDK.transfer({
```

Comment:
do we still have `transfer` in sdk? need to remove these references and in root README.md